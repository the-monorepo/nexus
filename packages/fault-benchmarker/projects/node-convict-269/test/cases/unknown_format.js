'use strict';

exports.conf = {
  unknownFormat: {
    format: 'unknownFormat',
    default: 'unknownFormat',
    doc: 'A value with an unknown format'
  }
};
