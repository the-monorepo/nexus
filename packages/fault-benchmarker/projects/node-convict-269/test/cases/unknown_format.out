'unknownFormat' uses an unknown format type: unknownFormat
